tools/sync_test_server/setup_mongodb_realm.sh

```diff
@@ -59,6 +59,14 @@ stitch-cli secrets add \
                   --base-url=http://localhost:9090 \
                   --config-path=/tmp/stitch-config
 
+#    - b) Firebase Cloud Messaging: create your own app and paste the "server key" here
+stitch-cli secrets add \
+                  --name="FCM" \
+                  --value="AAAAQLogIEA:APA91bGq5nvqcwR0NE3aQKADal14gP9sFS9MJ28JFqXeJ8h8kffHUaQ2GnpWFzbPRRCQn4P2Ux_HGTWJ9xxgZUmkv-Stcl4-Nbj3vcLOQP253wdzsQsc3dSIDKG53eWxTXWMr0_uleAE" \
```

Comment:
Is it safe to have these here, or should we move them to a local environment variable?

Reply:
I'm silly, I don't know why I didn't remove them. They were needed only for me to test the actual push. They aren't needed for testing the registration though. I will remove them.